Ginger/Ginger/Actions/ActionEditPages/VisualTesting/VRTComaprePage.xaml

```diff
@@ -50,7 +50,7 @@
         </DockPanel>
 
         <StackPanel Grid.Row="3" Orientation="Horizontal" Margin="0,10,0,0" x:Name="xVRTDiffTollerancePercent">
-            <Label  MinWidth="150px" Style="{StaticResource $LabelStyle}" Grid.Column="0" VerticalAlignment="Center" Content="Diff Tollerance Percent:"/>
```

Comment:
Please correct speelling of compare in this xaml filename

Reply:
Done